js/app.js

```diff
@@ -184,6 +184,11 @@ define(['jquery', 'knockout', 'ohdsi.util', 'appConfig', 'webapi/AuthAPI', 'atla
 							document.location = "#/welcome";
 						});
 					},
+          '/test-page': function () {
+            require(['test-page'], function () {
```

Comment:
Let's make a new foluder under /js called 'tests' and move our test components there. Then you can reference this componet as 'tests/test-page'

We should update all of our 'component' module refercts from 'moduleA' to 'components/moduleA' and then we can clean up all of the extra stuff in our requirejs config.